package.json

```diff
@@ -3,8 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@comicrelief/pattern-lab": "7.56.0",
-    "@comicrelief/storybook": "1.18.5",
+    "@comicrelief/pattern-lab": "^7.58.6",
+    "@comicrelief/storybook": "^1.18.6",
```

Comment:
I'm not sure if we should use the caret here. Just to avoid any pattern-lab/storybook updates going in in the future that haven't been tested in Giftaid. 
What do you think?

Reply:
as long as it hasn't been installed it shouldn't be a problem though.

Reply:
Please use locked versions